benchmark/addTypename.main.kts

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env kotlin
+@file:DependsOn("com.squareup.okio:okio:2.9.0")
+@file:DependsOn("com.squareup.moshi:moshi:1.11.0")
+
+import com.squareup.moshi.Moshi
+import okio.buffer
+import okio.sink
+import okio.source
+import java.io.File
+
+val adapter = Moshi.Builder().build().adapter(Any::class.java)
```

Comment:
what is this file? Is it might to be committed - it doesn't seem to be used in the actual benchmark from what I can tell?

Reply:
Good catch! It was a one-time script to add the `__typename` field to the [ZacSweers's largesample.json](https://github.com/ZacSweers/json-serialization-benchmarking/blob/main/models/src/main/resources/largesample.json) since GraphQL requires `__typename`. It's actually one area where we might gain a tiny bit of performance if we wanted to save this parsing for fields that are not polymorphic.
In all cases, this file isn't needed anymore, I removed it.